src/apis/stacks.js

```diff
@@ -72,10 +72,10 @@ export default (state) => {
    * @param  {mixed}   operations   array or single stack operation object
    * @return {Promise}
```

Comment:
please add above this line the param description for `options`. After that, run `npm run docs` and add the changed README.md file as well to the commit.